components/switch/switch-visibility.js

```diff
@@ -32,6 +41,35 @@ class VisibilitySwitch extends LocalizeCoreElement(SwitchMixin(LitElement)) {
 		return html`<d2l-icon icon="tier1:visibility-show"></d2l-icon>`;
 	}
 
+	get text() {
```

Comment:
I don't think we need documentation here anymore (i.e. [what we had before](https://github.com/BrightspaceUI/core/pull/2665/files#diff-43ec62628ec01309e32f82bffb749c81f3ef88699064f17e97180a4d261fed33L11-L14)) because we no longer allow the consumer to change this text anyway.

Reply:
Good question! Based on the search results for [d2l-switch-visibility](https://search.d2l.dev/search?project=Brightspace-Configs&project=BrightspaceHypermediaComponents&project=BrightspaceUI&project=BrightspaceUILabs&project=lms&full=%22d2l-switch-visibility%22&defs=&refs=&path=&hist=&type=&xrd=&nn=5) and [SwitchVisibility](https://search.d2l.dev/search?project=lms&full=%22SwitchVisibility%22&defs=&refs=&path=&hist=&type=&xrd=&nn=1), as well as searching for SwitchVisibility in the MVC and Testing solutions in Visual Studio, it looks like that was never used. 

In the case of backporting to MVC in the lms, if some place _was_ overriding the `text`, and we got rid of that within the wrapper class, would we expect an obvious error (like a compile-time error) to pop up?

Reply:
If the MVC control had a `text` property which you then removed, then yes there would likely be a compile time error (more likely from a full all than just a build of `lp` since it would likely be in an `le` project)

Reply:
I see. I also searched for `SwitchVisibility` in the `le` folder through Visual Studio and the only use case that came up wasn't making use of the `text` properly either

Reply:
Update: I _did_ find [a place or two (same use case) ](https://d2l.slack.com/archives/C03FC0GDVB7/p1658435070006839)where the Visibility Switch `text` property is being overridden.

Reply:
We should probably add a visual diff test that overrides `text` in `switch-visibility` (and then can remove that once it's no longer needed)

Reply:
Yeah I was gonna bring that up and ask if we would like to add tests for that (which I think makes sense, even though it's a bit weird since we're hoping to eventually remove this backwards-compatibility feature).